inc/theme-hooks.php

```diff
@@ -110,3 +110,63 @@ function alcatraz_output_footer_bottom() {
 		);
 	}
 }
+
+add_action( 'alcatraz_footer', 'alcatraz_output_social_network_icons', 80 );
+/**
+ * Output the social network icons.
+ *
+ * @since 1.0.0
+ */
+function alcatraz_output_social_network_icons() {
+
+	$options = get_option( 'alcatraz_options' );
+	?>
+
+	<div class="alcatraz-social-icon-wrap">
```

Comment:
I'm sure you know this, but you'll need a check here to see whether the option is set to output the icons before you output them. Also, I would output buffer this with ob_start and return ob_get_clean() below. If you return it instead of echoing it directly it'll be more flexible to use.